nifi-docker/dockerhub/sh/secure.sh

```diff
@@ -60,6 +60,7 @@ prop_replace 'nifi.web.http.host'   ''
 prop_replace 'nifi.web.https.port'  "${NIFI_WEB_HTTPS_PORT:-8443}"
 prop_replace 'nifi.web.https.host'  "${NIFI_WEB_HTTPS_HOST:-$HOSTNAME}"
 prop_replace 'nifi.remote.input.secure' 'true'
+prop_replace 'nifi.cluser.protocol.is.secure' 'true'
```

Comment:
There's a typo in the property name. It's 'cluser' but should be 'cluster'. Since this only applies for clusters I think it'd also make sense to set it to true only if it's a cluster node, so something like this (or an if...then):

```suggestion
prop_replace 'nifi.cluser.protocol.is.secure' "${NIFI_CLUSTER_IS_NODE:-false}"
```